plugins/rts54hub/rts54hub.quirk

```diff
@@ -118,3 +118,13 @@ Flags = updatable
 GType = FuRts54HubDevice
 FirmwareSizeMin = 0x10000
 FirmwareSizeMax = 0x20000
+
+#HP Portable USB-C Hub
+[USB\VID_03F0&PID_AE4A]
+Plugin = rts54hub
+Name = HP Portable USB-C Hub
+Flags = updatable
```

Comment:
I know the other quirk entries have this -- but is this line actually needed? It seems to unconditionally overwrite the flag set in `fu_rts54hub_device_setup()`

Reply:
> do you think I need to delete all the "Flags = updatable" in qurik file and re-push this patch

I think you could do a follow up patch that just removes all the "Flags = updatable` lines.

Reply:
narrator: _He did that_ -- PR updated.